Branch-SDK-Automation-TestBed/src/main/java/io/branch/saas/sdk/testbed/activities/MainActivity.java

```diff
@@ -82,6 +93,12 @@ public void onLinkCreate(String url, BranchError error) {
 
     @Override
     public void onClick(View view) {
+        if (view == btSetDMAParams) {
+            Branch.expectDelayedSessionInitialization(true);
+        } else {
+            Branch.getInstance().setIdentity("automation_test_user");
+        }
+        Branch.enableLogging();
```

Comment:
I've had to do this a few times in my own testing, I suggest moving this out to a custom application class and adding it there. Otherwise logs prior to this view will be lost

Reply:
If you enable here, do you disable elsewhere?